keywording/text.xml

```diff
@@ -29,7 +29,7 @@ A sample <c>KEYWORDS</c> entry might look like:
 </p>
 
 <codesample lang="ebuild">
-KEYWORDS="x86 sparc ~mips ~ppc ~ppc-macos -ia64"
+KEYWORDS="-ia64 ~mips ~ppc sparc x86 ~ppc-macos"
```

Comment:
Shouldn't `ppc-macos` come after `ppc` ?

Reply:
No, prefix keywords go last

Reply:
@gktrk As I said, no. Arch teams are using `ekeyword` and it works like this (random package example)

```sh
eb3eadc3b326 /usr/portage/app-editors/moe # ekeyword ~ppc moe-1.9.ebuild
moe-1.9: amd64 arm ~ppc x86 ~amd64-linux ~x86-linux
eb3eadc3b326 /usr/portage/app-editors/moe # ekeyword ~ppc-macos moe-1.9.ebuild
moe-1.9: amd64 arm ~ppc x86 ~amd64-linux ~x86-linux ~ppc-macos
eb3eadc3b326 /usr/portage/app-editors/moe # 
```